react-native-video.podspec

```diff
@@ -9,8 +9,8 @@ Pod::Spec.new do |s|
   s.description    = package['description']
   s.license        = package['license']
   s.author         = package['author']
-  s.homepage       = 'https://github.com/brentvatne/react-native-video'
-  s.source       = { :git => "https://github.com/brentvatne/react-native-video.git", :tag => "#{s.version}" }
+  s.homepage       = 'https://github.com/learnyst/react-native-video'
+  s.source       = { :git => "https://github.com/learnyst/react-native-video.git", :tag => "#{s.version}" }
```

Comment:
please change this for the PR

Reply:
Should we not just change to https://github.com/react-native-community/react-native-video? Don't know why brentvatne is still there

Reply:
Yes we need to change...I will change it

Reply:
Any idea when this fix will be complete?

Reply:
Hi, Any idea when a new release  that includes this fix will be made available?

Reply:
Hello, My group at work is using react-native-video in one of our apps and this issue is currently one of the blockers keeping us from releasing. Would you be able to provide an ETA on when the next version that includes this fix will be released?